go.mod

```diff
@@ -22,6 +22,7 @@ require (
 	github.com/agext/levenshtein v1.2.3 // indirect
 	github.com/apparentlymart/go-cidr v1.1.0 // indirect
 	github.com/apparentlymart/go-textseg/v13 v13.0.0 // indirect
+	github.com/apparentlymart/go-textseg/v15 v15.0.0 // indirect
```

Comment:
Why do we need two different versions of the same library?

Reply:
I am not really sure on this.
it got added as part of updating dependencies needed for v2 by running 
```
go get github.com/hashicorp/terraform-plugin-sdk/v2/plugin
```

https://github.com/ansible/terraform-provider-ansible/pull/67/commits/60c0ab7e31e7c94b79b9b156510a84d1add4f55e
@gravesm can I remove the v13.0.0 here, or can we keep it?

Reply:
Just run `go mod tidy`. This will ensure that `go.mod` matches what's in the code. In general, it's best to not try and hand edit `go.mod` or `go.sum`.

Reply:
Fixed! thank you